sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java

```diff
@@ -349,7 +353,7 @@ private Flux<ServiceBusMessageContext> getSession(Scheduler scheduler, boolean d
 
                 return new ServiceBusSessionReceiver(link, messageSerializer, connectionProcessor.getRetryOptions(),
                     receiverOptions.getPrefetchCount(), disposeOnIdle, scheduler, this::renewSessionLock,
-                    maxSessionLockRenewDuration);
+                    maxSessionLockRenewDuration, sessionIdleTimeout);
```

Comment:
just thinking loud. Now we've two parameters for ServiceBusSessionReceiver::Ctr - _disposeOnIdle_ and _sessionIdleTimeout_  related to idle-expiration. Will the code readability improve if we unify and pass only _sessionIdleTimeout_? as in

```java
final Duration sessionIdleTimeout;
if (disposeOnIdle) {
    sessionIdleTimeout = receiverOptions.getSessionIdleTimeout() != null 
         ? receiverOptions.getSessionIdleTimeout()
		 : connectionProcessor.getRetryOptions().getTryTimeout();
} else {
    sessionIdleTimeout = null;
}
new ServiceBusSessionReceiver(link, messageSerializer, connectionProcessor.getRetryOptions(),
    receiverOptions.getPrefetchCount(), sessionIdleTimeout, scheduler, this::renewSessionLock,
    maxSessionLockRenewDuration);
```

Reply:
good point, I also changed ReceiverOptions slightly to reduce amount of confusion around it. PTAL

Reply:
The ReceiverOptions's properties usage permutation was a bit of cognitive overload for me. Thank you for improving the readability!